examples/js/loaders/FBXLoader2.js

```diff
@@ -571,11 +571,10 @@
 
 							var faceVertexBuffer = [];
 							var polygonIndex = 0;
-							for ( var polygonVertexIndex = 0; polygonVertexIndex < indexBuffer.length; ++ polygonVertexIndex ) {
+							indexBuffer.forEach( function ( vertexIndex, polygonVertexIndex, indexBuffer ) {
```

Comment:
> The following is just my speculation.
We need to ask @mrdoob for the real reason.

Yep! It's for performance reasons.